providers/governor/streams/governor.go

```diff
@@ -288,10 +291,13 @@ func (g *jsGMgr) Start(ctx context.Context, name string) (Finisher, uint64, erro
 
 		res, err := jsm.ParsePubAck(m)
 		if err != nil {
-			if !jsm.IsNatsError(err, 10077) {
-				g.Errorf("Invalid pub ack: %s", err)
+			// jetstream sent us a puback error, this is retryable in the case of governors
+			if jsm.IsNatsError(err, 10077) {
```

Comment:
gvernor full is the healthy fail - if succesfully processed we know we can retry safely (its working, just telling us to try again) vs not working which is a fail.